packages/coinstac-ui/app/main/utils/boot/configure-main-hotswap.js

```diff
@@ -1,33 +0,0 @@
-const hotmop = require('hotmop');
-const _ = require('lodash');
-const logger = require('ampersand-app').logger;
-const app = require('ampersand-app');
-
-const swapLog = () => {
-  const logger = require('ampersand-app').logger; // eslint-disable-line global-require
-  logger.debug('swapped module');
-};
-
-const rebuildServiceApi = newModule => {
-  app.core = require('coinstac-client-core').get(); // eslint-disable-line global-require
```

Comment:
We never used this feature, nor am I sure it would have worked: _client-core_ maintains a fair amount of in-memory state.